lib/elements/elements.css

```diff
@@ -163,6 +163,11 @@
   /*box-shadow: 3px 3px 4px 0px rgba(15, 14, 14, 0.7);*/
 }
 
+#issues .issue.hasFix {
+  background: #444 url('../images/wrench.svg') right no-repeat;
```

Comment:
Hmm, or perhaps but the image here, and refer to it using a relative path from the codemirror editor's css?